src/filecoin/FilecoinEngine.ts

```diff
@@ -384,6 +431,38 @@ export class FilecoinEngine extends CurrencyEngine<
     } while (messagesChecked < messageCount)
   }
 
+  filfoxMessageToEdgeTransaction = (
+    message: FilfoxMessage
+  ): EdgeTransaction => {
+    const addressString = this.address.toString()
+    let netNativeAmount = message.value
+    const ourReceiveAddresses = []
+
+    const networkFee = '0' // TODO: calculate transaction fee from onchain gas fields
```

Comment:
Yikes. Major todo. Can you create a task to patch this up in another PR along with Filscan.